magento/magento2
# If stock is bellow OutOfStock Threshold, a negative qty is displayed in Product List Page

### Description (*)

If stock is bellow OutOfStock Threshold, a negative qty is displayed in Product List Page
### Fixed Issues (if relevant)

1. magento/magento2#9130: If stock is bellow OutOfStock Threshold, a negative qty is displayed in Product List Page

### Manual testing scenarios (*)

1. Set qty to 5 for a product, and Out of Stock threshold to 10. Save the product
When browsing that product in the frontend a "Only -5 left" is displayed under "Out of Stock" message.
2. Remove the block if qty is negative.

### Contribution checklist (*)
